hoomd/mpcd/pytest/test_collide.py

```diff
@@ -594,3 +594,130 @@ def test_rigid_collide_free_participatory(
 
             expected_velocity = np.add(expected_tangential_velocity, new_velo_central)
             assert np.allclose(new_velo_constituent, expected_velocity)
+
+    def test_rigid_collide_central_rescale(
```

Comment:
Is this test checking somewhere that linear momentum is conserved? It seems like it does not or it would fail based on the edits I made to the code above... maybe I am mistaken.